### PR TITLE
Fixes #9236: Log levels should be incremental

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/9236

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

File: tree/10_ncf_internals/configuration.cf [hunk 1]
```diff
@@ -63,6 +63,14 @@ bundle agent configuration
       "pass2" expression => "pass1";
       "pass1" expression => "any";
 
+      # Make log level incremental
```
Comment (peckpeck) [inline tree/10_ncf_internals/configuration.cf]: Missing the global scope here

File: tree/10_ncf_internals/configuration.cf [hunk 2]
```diff
@@ -63,6 +63,14 @@ bundle agent configuration
...
+      "debug" expression => "trace";
+      # CFEngine's stdlib verbose mode
+      "verbose_mode" expression => "debug";
+      "info" expression => "debug";
+      # CFEngine's stdlib inform mode
+      "inform_mode" expression => "info";
```
Comment (peckpeck) [inline tree/10_ncf_internals/configuration.cf]: You should add the stdlib classes

**Comment by amousset** [comment]
Commit modified

**Comment by peckpeck** [comment]
I think we should retarget to v0.x

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

File: tree/10_ncf_internals/configuration.cf [hunk 3]
```diff
@@ -59,6 +59,18 @@ bundle agent configuration
...
+      "verbose_mode" expression => "debug",
+                          scope => "namespace";
+      "info"         expression => "debug",
+                          scope => "namespace";
+      # CFEngine's stdlib inform mode
+      "inform_mode"  expression => "info" ,
+                          scope => "namespace";
+
   # We cannot use the generic method to expand template, as it relies on the logger,
   # which is not yet defined
```
Comment (amousset) [inline tree/10_ncf_internals/configuration.cf]: Test comment

**Comment by amousset** [comment]
PR replaced by https://github.com/Normation/ncf/pull/532

**Pull request closed without merging**